lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1721,6 +1725,28 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
   }
 
   void calendarEventSuccess(CalendarEventReplySuccess success) {
+    final eventReplyResponse = success.calendarEventReplyResponse;
+    log('SingleEmailController::calendarEventSuccess:EventReplyResponse = $eventReplyResponse');
+    if (eventReplyResponse is CalendarEventAcceptResponse
+        && (eventReplyResponse.accepted == null
+            || eventReplyResponse.accepted?.isEmpty == true)) {
```

Comment:
why not handle it in `interactor` or `datasource`?

Reply:
Sounds reasonable. Fixed.